README.md

```diff
@@ -30,13 +30,21 @@ The response is a JSON with the following format:
 }
 ```
 
----
+### API key and API secret
+
+If you are an EDMdesigner intergator then you already have them.
```

Comment:
This is not entirely true. They don't have a separate api key for the generator. They have an api key for the edm api, and the edm api uses the generator with its api key. (so basically they use the edmdesigner api key)